package.json

```diff
@@ -27,7 +27,9 @@
   "homepage": "https://github.com/UKHomeOffice/asl#readme",
   "dependencies": {
     "@asl/components": "^7.2.0",
-    "@asl/pages": "^18.19.2",
+    "@asl/constants": "^0.6.2",
+    "@asl/dictionary": "^1.1.5",
```

Comment:
What's the reasoning for adding the extra dependencies?

If they're not used directly in the code in this repo then they shouldn't be defined in the package.json as doing so will make it harder to dedupe in future.